index.js

```diff
@@ -52,11 +52,16 @@ const generateDurationString = (totalSeconds) => {
   return `PT${hoursString}${minutesString}${secondsString}`;
 }
 
-const generateManifest = (data) => {
+const generateManifest = (data, is_audio = false) => {
   const { duration, ext, format_id, fragments, fragment_base_url } = data;
   const durationString = generateDurationString(duration);
+  const type = is_audio ? 'audio' : 'video';
   let time = 0;
```

Comment:
Can't comment on the relevant line, and it's not your problem to fix either, but I can't believe Andrew let line 87 exist... side effects inside a `.map`